mitiq/benchmarks/tests/test_ghz_circuits.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) 2022 Unitary Fund
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Tests for GHZ circuits."""
+
+import pytest
+import numpy as np
+
+from mitiq.benchmarks import ghz_circuits
+from mitiq._typing import SUPPORTED_PROGRAM_TYPES
+
+
+@pytest.mark.parametrize("nqubits", [1, 3, 5, 10])
```

Comment:
You can safely get away with just two values of `nqubits` here, say `1` and `5`.